modules/hostdb/hostentry_test.go

```diff
@@ -0,0 +1,74 @@
+package hostdb
+
+import (
+	"testing"
+
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// TestHostWeight probes the hostWeight function.
+func TestHostWeight(t *testing.T) {
+	hdbt := newHDBTester("TestHostWeight", t)
+
+	// Create two identical entries, except that one has a price that is 2x the
+	// other. The weight returned by hostWeight should be 1/8 for the more
+	// expensive host.
+	entry1 := hostEntry{
+		HostSettings: modules.HostSettings{
+			Price: types.NewCurrency64(3),
+		},
+	}
+	entry2 := hostEntry{
+		HostSettings: modules.HostSettings{
+			Price: types.NewCurrency64(6),
+		},
+	}
```

Comment:
since `HostSettings` is embedded, you could alternatively write:
```go
var entry1, entry2 hostEntry
entry1.Price = types.NewCurrency64(3)
entry2.Price = types.NewCurrency64(6)
```